package.json

```diff
@@ -116,8 +127,10 @@
 		"babel-plugin-transform-vue-jsx": "^3.5.1",
 		"babel-preset-env": "^1.6.1",
 		"babel-preset-stage-2": "^6.22.0",
+		"babel-preset-vue": "^2.0.2",
```

Comment:
Why are we adding this?

Reply:
Looks like it's a peer dependency of Vue Storybook https://storybook.js.org/docs/guides/guide-vue/